lib/node_modules/@stdlib/utils/is-uppercase/README.md

```diff
@@ -0,0 +1,71 @@
+# isUppercase
+
+> Test if a value is an uppercase string.
+
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var isUppercase = require( '@stdlib/utils/is-uppercase' );
+```
+
+#### isUppercase( value )
+
+Tests if a `value` is an uppercase `string`.
+
+``` javascript
+var bool = isUppercase( 'HELLO' );
+// returns true
+
+bool = isUppercase( 'salt and light' );
+// returns false
+```
+
+</section>
+
+<!-- /.usage -->
+
+
+<section class="notes">
+
+## Notes
+
+* This module first validates that a `value` is a `string`. For all other types, the function returns `false`.
+
+</section>
+
+<!-- /.notes -->
+
+
+<section class="examples">
+
+## Examples
+
+``` javascript
+var isUppercase = require( '@stdlib/utils/is-uppercase' );
+
+var bool = isUppercase( 'HELLO' );
+// returns true
+
+bool = isUppercase( '' );
```

Comment:
Refactored the implementation. The function now checks not only that the input string is equal to the uppercased string, but also different from the lowercased string. This way, it returns `false` for strings consisting only of say punctuation as well as the empty string. It's not anymore possible for both functions to return `true` for a given `string`. This seems to be in line with how analogous functions in other languages like Java behave.